lib/tasks/link_governments.rake

```diff
@@ -0,0 +1,13 @@
+namespace :governments do
+  task relink: :environment do
+    Edition.where(state: %w[draft published]).each do |edition|
```

Comment:
```suggestion
    Edition.where(state: %w[draft published]).find_each do |edition|
```

Using `find_each` can improve performance as it avoids needing to instance a new object for each record (the same instance is re-used multiple times).